### ISSUE TITLE
Get stuck while sign in (Class not found when unmarshalling: com.google.android.gms.auth.api.signin.internal.SignInConfiguration)

### ISSUE THREAD

**Issue by glacier9698** [playgameservices/play-games-plugin-for-unity#2992]
Hello. 
I faced the problem of infinite loading when login to GPGS.

Here is android log.
![bug](https://user-images.githubusercontent.com/77633560/105044583-3ed4b680-5aaa-11eb-9d49-33503280bebb.PNG)
![오류](https://user-images.githubusercontent.com/77633560/104942527-e8a73b00-59f7-11eb-8d0a-f9e8a1eda912.PNG)

I searched Google and found many solutions about similar type of problems.
But this is little different.

![text](https://user-images.githubusercontent.com/77633560/104943338-f5785e80-59f8-11eb-8738-11676d07fbf1.PNG)

1. Checked 'Custom Proguard File' in unity project setting.
2. Put above in proguard-user.txt
3. Done 'Force Resolve'
4. Set Oauth2.0 SHA-1 (both app, upload correctly)
5. App is released as 'Internal test'
6. Both Unity build and download from store are have infinite loading

Unity Version   : 2020.1.14f1
Plugin Version : v10.12

Please give me any idea..
Thanks !

**Comment by glacier9698** [comment]
I checked my unity proejct Sha1 fingerprint in command prompt,  and it was completely same as the play console.
![check it](https://user-images.githubusercontent.com/77633560/105045035-c6222a00-5aaa-11eb-849c-4b7aa877269c.PNG)

**Comment by glacier9698** [comment]
Okay finally i found it.

I changed the 'Testing' to 'In Production'
![app](https://user-images.githubusercontent.com/77633560/105170462-4f476880-5b60-11eb-97d2-6f48c744c8e8.PNG)

But i really wonder why I couldn't login with the registered 'TestID' when it was 'Testing' step.
I checked several times whether the test ID was entered correctly.
Does it take time to renew?
or am I wrong about something?
